client/src/main/java/com/vesoft/nebula/entry/AbstractNebulaCallback.java

```diff
@@ -19,24 +19,28 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(AbstractNebulaCallback.class);
 
-    protected TBase result;
+    protected volatile TBase result;
```

Comment:
maybe `volatile` is unnecessary

Reply:
result will be updated by SelectThread, which may not be visible for current thread

Reply:
When the RPC is finished, it's to call the callback. Using `volatile` to decorate the result will make sure it's be found immediately?

Reply:
Yes. more details: [What does volatile do?](https://www.cs.umd.edu/~pugh/java/memoryModel/jsr-133-faq.html#volatile)